colors/embark.vim

```diff
@@ -94,7 +94,7 @@ function! s:maybe_italic(style)
   if g:embark_terminal_italics == 1
     " if there are other styles than italics concat
     if empty(a:style)
-      return s:maybe_italic("")
```

Comment:
This triggers  an infinite recursion, as `empty(a:style)` will always be empty at this point.